pytition/petition/views.py

```diff
@@ -74,6 +74,11 @@ def index(request):
             user = get_session_user(request)
         else:
             user = request.user
+        all_petitions = Petition.objects.filter(published=True).order_by('-id')
+        paginator = Paginator(all_petitions, 12)
```

Comment:
Just one thing, i would put the `12` value either in a constant or in Django settings if it's supposed to be modifiable.

Reply:
ah right, good idea!
I'd go for the Django settings `in base.py`

Reply:
Good idea, I'll add it to the base settings